wikipedia_for_humans/__init__.py

```diff
@@ -14,8 +14,8 @@
 def _get_page(page_name, lang="en", strict=False, auto_disambiguate=False):
     if isinstance(page_name, wikipediaapi.WikipediaPage):
         return page_name
-    wiki_wiki = wikipediaapi.Wikipedia(lang)
-    wiki_wiki._session = CachedSession(backend="memory", expire_after=60, headers=HEADERS)
+    wiki_wiki = wikipediaapi.Wikipedia(HEADERS, lang)
```

Comment:
Consider using kwargs to avoid this kind of thing in the future:

```python
wiki_wiki = wikipediaapi.Wikipedia(user_agent=HEADERS["User-Agent"], language=lang)

```

Reply:
Also it would appear that `HEADERS` is a set, not a dict...is that deliberate?